docs/user_guide/storing/index_hnswlib.md

```diff
@@ -208,4 +208,7 @@ To delete nested data, you need to specify the `id`.
 del doc_index[index_docs[6].id]
 ```
 
-Check [here](docindex#nested-data-with-subindex) for nested data with subindex.
\ No newline at end of file
+Check [here](docindex#nested-data-with-subindex) for nested data with subindex.
+
+### Update elements
+In order to update a Document inside the index, you only need to reindex it with the updated attributes.
```

Comment:
can we have a code example ? It is unlcear to me what this means

Reply:
okay looking at the test make it clearer. But I guess a code snipet could help as well